cfme/common/host_views.py

```diff
@@ -302,6 +302,18 @@ def is_displayed(self):
         return self.in_compute_infrastructure_hosts and self.title.text == 'Credentials/Settings'
 
 
+class HostsCompareView(ComputeInfrastructureHostsView):
+    """Compare Host / Node page."""
+
+    @property
+    def is_displayed(self):
+        title = "Compare Host / Node"
+        return (self.logged_in_as_current_user and
+                # self.navigation.currently_selected == ['Compute', 'Infrastructure',
```

Comment:
Q: why are these lines commented out? You should be able to check the currently selected navigation and the title text.

Reply:
I am removing this code. It does not work without supporting code for navigation. It may need to be added later as I develop tests for this FA.

Reply:
I added this check as suggested. I was initially confused o when it could be used.